prompt_engine.py

```
@@ -0,0 +1,365 @@
+            # Parse vibe attributes if available
+            if 'vibe_attributes' in row and pd.notna(row['vibe_attributes']):
+                try:
+                    if isinstance(row['vibe_attributes'], str):
+                        vibe_attrs = json.loads(row['vibe_attributes'])
+                    else:
+                        vibe_attrs = row['vibe_attributes']
+                    
+                    positive_vibes = [k for k, v in vibe_attrs.items() if v]
+                    if positive_vibes:
+                        lines.append(f"Atmosphere: {', '.join(positive_vibes)}")
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)